### DISCUSSION TITLE
Is it any way to customize themes? Right now is disabled, but I ask anyways if there's any chance...

### DISCUSSION THREAD

**Discussion by Tobarrientos2** [mitchmac/ServerlessWP#34] [Q&A] [answered]
I want to customize some info about the site and the theme.
Obviously when i attempt to customize the site I can't because "I need a higher level of permission."

Thanks anyways!

**Comment by mitchmac** [comment]
Hi @Tobarrientos2 -- this is a common question so I've posted [a note](https://github.com/mitchmac/ServerlessWP/discussions/35) that I think answers what you are looking for. The short version is: themes, plugins and their customized files need to be added to your git repository.

**Reply by Tobarrientos2** [reply, marked as answer]
I didn't really mean that! What happened is that I installed a theme and a plugin. In order to modify the theme, I needed to customize it, customize the site's appearance. That option is the one where you can also change the site's thumbnail. I didn't know how to do it until I found this plugin (https://wordpress.org/plugins/user-role-editor/), with which I was able to assign myself all the capabilities as an administrator. For some reason, I didn't have many permissions as an administrator before. Nevertheless, everything is running perfectly now! I even gave myself permissions to install plugins, attached a screenshot as well. 
Thanks a lot! Anyways :) Serverless is awesome

**Reply by mitchmac** [reply]
Woohoo!
